odk1-src/form-question-types.rst

```diff
@@ -1486,6 +1486,23 @@ Provides the user a drawing pad and collects the drawn image.
 
   image,draw_image_widget,Draw widget ,draw,image type with draw appearance
   
+Scaling down images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Images created with any of the image widgets described above can be automatically scaled down on save by using the ``max-pixels`` parameter. If the long edge of the image is larger than the maximum size specified, the image is resized proportionally so that the long edge matches the provided pixel value. This is useful to reduce the upload size when bandwidth is limited. 
+
+Available in Collect since v1.10.0 and in XLSForm since 7/2018.
```

Comment:
Versioning XLSForm around dates feels odd. Is the date when it was in the spec? When it was in pyxform? When it was in XLSForm online? Offline?

Reply:
I don't think we have any better option. It's the date when it started being supported across all those tools.